clients/src/main/java/org/apache/kafka/common/utils/Utils.java

```diff
@@ -1271,4 +1274,28 @@ private static byte checkRange(final byte i) {
         }
         return map;
     }
+
+    public static long getDateTime(String timestamp) throws ParseException {
```

Comment:
Since this is a public function, it would be good to add a javadoc. Please refer to the function above in the same class for example.